### PR TITLE
Tests for allies notifications

### PR DESCRIPTION
**Pull request by chrisrp**
* Adds simplecov
* Adds tests for allies notification
* minor refactoring on notification_mailer

To check the coverage run the tests and check the generated folder:
`coverage/index.html`

### CODE REVIEW

File: app/mailers/notification_mailer.rb [hunk 1]
```diff
@@ -1,166 +1,161 @@
 class NotificationMailer < ApplicationMailer
```
Comment (julianguyen) [inline app/mailers/notification_mailer.rb]: Unrelated to this PR, but we should move the text in `en.yml`

**Comment by julianguyen** [comment]
LGTM great work @chrisrp :D
Also! Please add yourself to the contributors page :D

File: app/mailers/notification_mailer.rb (hunk 1 above)
Comment (chrisrp) [inline app/mailers/notification_mailer.rb]: I thought the same.. once we have the entire class covered I will try to improve this.

Comment (julianguyen) [inline app/mailers/notification_mailer.rb]: That would be fantastic! Thanks Chris :)

**Pull request merged**